libs/shared/data/src/lib/form-builder.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { FormlyFieldConfig, FormlyFormOptions } from '@ngx-formly/core';
```

Comment:
In general, unless you're making your own interfaces and mappers between these types, I don't see any benefit of this file and IMO we should use formly models directly